### PR TITLE
Fibonacci homework

### PR DESCRIPTION
**Pull request by MoreThanADC**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @MoreThanADC

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+
+    int result {0};
+    int previous {0};
+    int actual {1};
+
+    if (sequence == 0) {
+        return 0;
+    } else if (sequence == 1) {
+        return 1;
+    } else {
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:13]: If you return variable here, you don't need to use `else if`

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
-    return 0;
+
+    int result {0};
+    int previous {0};
+    int actual {1};
+
+    if (sequence == 0) {
+        return 0;
+    } else if (sequence == 1) {
+        return 1;
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:12]: Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

File: homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
-}
+   
+    if (sequence == 0) {
+        return 0;
+    } else if (sequence == 1) {
+        return 1;
+    } else {
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:29]: Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

Also `else if` and `else` is redundant.

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**